rollback1.py

```diff
@@ -26,96 +39,171 @@ def create_contract(self, data):
             dataC = data
         res = self.call("NewContract", dataC)
         return name, code
-        
+
+    def addNotification(self):
+        # create contract, wich added record in notifications table
+        function = """DBInsert("notifications", "recipient->member_id,notification->type,notification->header,notification->body", "-8399130570195839739", 1, "Message header", "Message body")"""
+        body = """ { \n data {}	\n conditions {} \n	action { \n """ + function + """ \n } }"""
+        code, name = utils.generate_name_and_code(body)
+        data = {"Wallet": '', "ApplicationId": 1,
+                "Value": code,
+                "Conditions": "ContractConditions(`MainCondition`)"}
+        res = self.call("NewContract", data)
+        self.assertGreater(int(res["blockid"]), 0, "BlockId is not generated: " + str(res))
+        # change permission for notifications table
+        permission = """{"insert": "true","update" : "true","new_column": "true"}"""
+        dataEdit = {"Name": "notifications", "Permissions": permission}
+        res = self.call("EditTable", dataEdit)
+        # call contract, wich added record in notification table
+        res = self.call(name, "")
+        # change permission for notifications table back
+        permission = """{"insert": "ContractAccess(\\"Notifications_Single_Send_map\\",\\"Notifications_Roles_Send_map\\")", "update": "ContractConditions(\\"MainCondition\\")", "new_column": "ContractConditions(\\"MainCondition\\")"}"""
+        dataEdit = {"Name": "notifications", "Permissions": permission}
+        res = self.call("EditTable", dataEdit)
+
+    def getCountTable(self,name):
+        return utils.getCountTable(host, db, login, pas, name)
```

Comment:
maybe its better just pass db config to functions like that, so helper function getCountTable is useless
utils.getCountTable(self.conf)